resume-buddy/src/main/java/com/google/sps/data/Reviewee.java

```diff
@@ -1,15 +1,32 @@
 package com.google.sps.data;
 
-/*TODO: add more information from form input into Reviewee*/
 public final class Reviewee {
 
   private final String firstName;
   private final String lastName;
   private final String email;
+  private final String school;
+  private final String year;
+  private final String career;
+  private final String degreePref;
+  private final String numYearsPref;
```

Comment:
Instead of Strings, these should be using the enums you just created:

```
private final NumYears year;
private final Career career;
private final Degree degreePreference;
private final NumYears numYearsPreference;
```